ensembl/conf/json/bos_taurus_vcf.json

```diff
@@ -3,9 +3,9 @@
     {
       "id": "nextgen_cow_irbt",
       "species": "bos_taurus",
-      "assembly": "ARS-UCD1.2",
+      "assembly": "ARS-UCD1.3",
       "type": "local",
-      "filename_template": "/bos_taurus/ARS-UCD1.2/variation_genotype/IRBT_ARS-UCD1_2.vcf.gz",
```

Comment:
Shouldn't the path still include the 'variation_genotype'? It does for the other species.